daemon/mgr/spec_mount.go

```diff
@@ -40,10 +41,7 @@ func clearReadonly(m *specs.Mount) {
 	m.Options = opts
 }
 
-// setupMounts create mount spec.
-func setupMounts(ctx context.Context, c *Container, s *specs.Spec) error {
-	var mounts []specs.Mount
-	// Override the default mounts which are duplicate with user defined ones.
+func overrideDefaultMount(mounts []specs.Mount, c *Container, s *specs.Spec) ([]specs.Mount, error) {
```

Comment:
Maybe we can add some test cases for the new function.

Reply:
It isn't complicated logic, I think it no need to add unit test.

Reply:
Should add, but can be in another pr